cms/tests/test_alias.py

```diff
@@ -92,6 +92,44 @@ def test_alias_recursion_across_pages(self):
             self.assertEqual(response.status_code, 200)
             self.assertContains(response, '<div class="info">', html=False)
 
+    def test_alias_recursion_across_page_translation(self):
+        page = api.create_page(
+            "Alias plugin",
+            "col_two.html",
+            "en",
+            slug="page_1",
+            published=True,
+            in_navigation=True,
+        )
+
+        ph_1 = page.placeholders.get(slot="col_left")
+        ph_2 = page.placeholders.get(slot="col_sidebar")
+
+        api.add_plugin(ph_1, 'StylePlugin', 'en', tag_type='div', class_name='info_1')
+        source_plugin = api.add_plugin(ph_2, 'StylePlugin', 'en', tag_type='div', class_name='info_2')
+
+        api.create_title("fr", "page_1", page)
+
+        alias_placeholder = api.add_plugin(ph_1, 'AliasPlugin', 'fr', alias_placeholder=ph_1)
+        alias_plugin = api.add_plugin(ph_2, 'AliasPlugin', 'fr', plugin=source_plugin)
+        
+        page.publish('fr')
+
+        self.assertTrue(alias_placeholder.is_recursive())
+        self.assertTrue(alias_plugin.is_recursive())
+
+        with self.login_user_context(self.get_superuser()):
+            response = self.client.get(page.get_absolute_url(language='fr') + '?preview&edit_off')
+            self.assertEqual(response.status_code, 200)
+            self.assertContains(response, '<div class="info_2">', html=True)
```

Comment:
At this point, in published rendering mode, the test with an alias plugin passes and after with an alias placeholder does not.